iaso/tests/api/test_profile_logs.py

```diff
@@ -0,0 +1,542 @@
+                        },
+                    }
+                ],
+            )
+        # by user 2 for editable user 2 with org unit 1 after date
+        with patch("django.utils.timezone.now", lambda: date2):
+            cls.log_6 = Modification.objects.create(
+                user=cls.user_with_permission_2,
+                object_id=str(cls.edited_user_1.iaso_profile.id),
+                source="API",
+                content_type=cls.content_type,
+                past_value=[
+                    {
+                        "pk": cls.edited_user_2.iaso_profile.id,
+                        "fields": {
+                            "user": cls.edited_user_2.id,
+                            "email": "",
+                            "dhis2_id": "12345",
+                            "account": cls.account.id,
+                            "language": "fr",
+                            "projects": [cls.project_1.id],
+                            "org_units": [cls.org_unit_1.id],
+                            "user_roles": [],
+                            "user_permissions": ["iaso_fictional_permission"],
+                            "phone_number": "+32485996633",
+                            "username": cls.edited_user_2.username,
+                            "first_name": "",
+                            "last_name": "",
+                            "deleted_at": None,
+                            "home_page": "",
+                        },
+                    }
+                ],
+                new_value=[
+                    {
+                        "pk": cls.edited_user_2.iaso_profile.id,
+                        "fields": {
+                            "user": cls.edited_user_2.id,
+                            "email": "",
+                            "dhis2_id": "12345",
+                            "account": cls.account.id,
+                            "language": "fr",
+                            "projects": [cls.project_1.id],
+                            "user_roles": [],
+                            "org_units": [cls.org_unit_1.id],
+                            "user_permissions": ["iaso_fictional_permission"],
+                            "phone_number": "+32485996633",
+                            "username": cls.edited_user_2.username,
+                            "first_name": "Kung",  # Changed
+                            "last_name": "Lao",  # Changed
+                            "deleted_at": None,
+                            "home_page": "",
+                            "password_updated": False,
+                        },
+                    }
+                ],
+            )
+
+    def test_user_no_permission(self):
+        """GET /api/userlogs/ without USERS_ADMIN permission --> 403"""
+        self.client.force_authenticate(self.user_without_permission)
+        response = self.client.get("/api/userlogs/")
+        self.assertJSONResponse(response, 403)
```

Comment:
```suggestion
        self.assertJSONResponse(response, status.HTTP_403_FORBIDDEN)
```

`from rest_framework import status` for the import

And same comment for the other codes